corehq/apps/userreports/models.py

```diff
@@ -1273,7 +1273,8 @@ def _raise_not_found():
                 _raise_not_found()
         else:
             try:
-                config = get_document_or_not_found(DataSourceConfiguration, domain, config_id)
+                config = get_document_or_not_found(DataSourceConfiguration, domain, config_id,
+                                                   additional_doc_types=[RegistryDataSourceConfiguration.__name__])
```

Comment:
this will return a `DataSourceConfiguration` instead of a `RegistryDataSourceConfiguration` which does work for the changes in this PR but I think it would be better to subclass `ReportConfiguration` and override the `config` property to fetch the correct doc type. We'll likely need to subclass ReportConfiguration for other reasons anyway.

Reply:
I think you can undo this change now right?

Reply:
ah yes thank you

Reply:
@snopoke ✅  3c2828e